packages/usa-nav/src/styles/_usa-nav.scss

```diff
@@ -209,20 +209,26 @@ $expand-less-icon: map-merge(
       }
     }
 
-    &[aria-expanded="false"] {
+    &[aria-expanded] {
```

Comment:
The diff here is a bit more complex than originally planned since I initially didn't notice that the positioning styles were largely duplicated across `&[aria-expanded="false"]` and `&[aria-expanded="true"]` selectors. I could have duplicated the `top` and `transform` styles for the second style selector, but I figured it'd be more maintainable (avoid similar such confusion in the future) to consolidate.